webui/react/src/hooks/usePermissions.ts

```diff
@@ -171,7 +170,7 @@ const relevantPermissions = (
     .forEach((r) => {
       // TODO: is it possible a role is assigned to this workspace,
       // but not all of its permissions?
-      permissions = permissions.concat(r.permissions.filter((p) => p.isGlobal || workspaceId));
+      permissions = permissions.concat(r.permissions);
```

Comment:
Had to remove filter because some permission can both be global and non-global, like `PERMISSION_TYPE_ASSIGN_ROLES`, if we only take into account global only permissions, on a global level, permission like `canGetPermissions` will never be true.

Reply:
I think this is an opportunity to remove `canGetPermissions` altogether, because it is a copy of canAssignRoles, and once it is removed from `useModalCreateUser.tsx` we don't use it anywhere else

Reply:
I agree with @mapmeld removing `canGetPermissions` here makes sense. Then this LGTM!